src/plugin.js

```diff
@@ -91,12 +91,17 @@ export default function ({ types: t }) {
         // TODO add other webpack checks
         if (config.module.loaders.some((l) => l.test.test(args[0].value))) {
           const [{ value: filePath }] = args;
+          const isRelativeFilePath = /^\.\.?[\/]/.test(filePath);
 
-          const fileAbsPath = resolve.sync(
+          const resolver = ResolverFactory.createResolver({
+            fileSystem: new SyncNodeJsInputFileSystem(),
+            ...config.resolve,
+          });
+
+          const fileAbsPath = resolver.resolveSync(
+            {},
+            isRelativeFilePath ? dirname(filenameRelative) : process.cwd(),
```

Comment:
This enables the following situations:

*index.js*
```js
import b from 'b.js'; // <-- b.js is in the root of the project (where the plugin is being run)
import c from 'src/c.js'; // <-- src/ directory is also in the root of the project
```

In these cases the webpack configuration can be empty, since the current directory of the process is searched in the plugin when the file path is not relative.

---

However, I'm having second thoughts about this feature. It can also be achieved by configuring `resolve.modules` to include the current directory, like this:

*webpack config*
```js
{
  "resolve": {
    "modules": __dirname  // or pass an array
  }
}
```